core/appeal/service.go

```diff
@@ -276,7 +276,7 @@ func (s *Service) Create(ctx context.Context, appeals []*domain.Appeal, opts ...
 		appeal.Policy = nil
 
 		for _, approval := range appeal.Approvals {
-			if approval.Index == len(appeal.Approvals)-1 && approval.Status == domain.ApprovalStatusApproved {
+			if (approval.Index == len(appeal.Approvals)-1 && approval.Status == domain.ApprovalStatusApproved) || appeal.Status == domain.AppealStatusApproved {
```

Comment:
can we add test to this?

Reply:
updated in the existing test case itself by adding a manual approval step which is skipped